Automattic/vip-cli
# Use official Elasticsearch image on dev-env

## Description

This PR switches the Elasticsearch image that's being used in the dev environment, from Bitnami's to the official one. The biggest difference being that the official image supports ARM architectures. We are also bumping the default ES version to `7.10.1`.

## Steps to Test

1. Check out PR.
2. Create a new environment.
3. Verify that the environment and vip-search work as expected.
